pulp_rpm/tests/functional/api/test_consume_content.py

```diff
@@ -26,30 +30,49 @@
 class PackageManagerConsumeTestCase(unittest.TestCase):
     """Verify whether package manager can consume content from Pulp."""
 
-    def test_all(self):
+    @classmethod
+    def setUpClass(cls):
+        """Verify whether dnf or yum are present."""
+        cls.cfg = config.get_config()
+        cls.pkg_mgr = cli.PackageManager(cls.cfg)
```

Comment:
This whole try can be replaced by:

```py
cls.pkg_mgr.raise_if_unsupported(unittest.SkipTest, 'This test requires dnf or yum.')
```

https://github.com/PulpQE/pulp-smash/blob/master/pulp_smash/cli.py#L773